app/code/community/Ebizmarts/MailChimp/etc/system.xml

```diff
@@ -249,7 +257,8 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                             <can_be_empty>1</can_be_empty>
-                            <comment><![CDATA[If you enable this Magento confirmation email will be sent for subscribers instead of Mailchimp email.]]></comment>
+                            <comment>
```

Comment:
This blank lines after the start tag cause failure.

Reply:
Solved.